modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/import/edit_batch_planner_plan.jsp

```diff
@@ -62,6 +62,32 @@ EditBatchPlannerPlanDisplayContext editBatchPlannerPlanDisplayContext = (EditBat
 								/>
 							</div>
 
+							<div class="alert alert-autofit-stacked alert-indicator-start alert-info alert-inline">
```

Comment:
In portal there are occurrences of a `clay:alert` taglib that should do what you're doing here 

[Here's an example ](https://github.com/liferay/liferay-portal/blob/54b94f08e29c1dbcd4d72446fa3906a171ab0843/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp#L121)